### PR TITLE
adding partition recognition support

### PR DESCRIPTION
**Pull request by dgraeber**
*Issue #, if available:*
#108 

*Description of changes:*
Adding resolution of aws partition based on session of cli invocation.  Removed hard-coded partition from templates and code. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws_codeseeder/services/_utils.py [hunk 1]
```diff
@@ -73,8 +73,30 @@ def get_region(session: Optional[Union[Callable[[], boto3.Session], boto3.Sessio
...
+    Returns
+    -------
+    Tuple[str, str, str]
+        returns the account id, role arn, and aws partition of the session provided
+    """
+    sts_info = boto3_client(service_name="sts", session=session).get_caller_identity()
+    return (sts_info.get("Account"), sts_info.get("Arn"), sts_info.get("Arn").split(":")[1])
+
+
+# def get_account_id(session: Optional[Union[Callable[[], boto3.Session], boto3.Session]] = None) -> str:
```
Comment (srinivasreddych) [inline aws_codeseeder/services/_utils.py:94]: Can you clean up these lines?

**Comment by srinivasreddych** [comment]
Out of curiosity, are you exposing the method of grabbing partition from codeseeder, so you can use re-use in seedfarmer?

File: aws_codeseeder/services/_utils.py (hunk 1 above)
Comment (dgraeber) [reply aws_codeseeder/services/_utils.py:94]: yes

**Comment by dgraeber** [comment]
@srinivasreddych `Out of curiosity, are you exposing the method of grabbing partition from codeseeder, so you can use re-use in seedfarmer?`

No - seedfarmer has it's own utility method for resolving session-related calls

**Review by srinivasreddych — APPROVED** [review]

**Pull request merged**